.github/workflows/sdk.yml

```diff
@@ -0,0 +1,89 @@
+name: Example SDK Php
+on:
+  pull_request:
+    branches: ["master"]
+  workflow_dispatch:
+
+jobs:
+  static:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_DB: progressively
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v2
+      - name: Clone  repo
+        run: git clone "https://github.com/progressively-crew/progressively.git" temp/ && mv temp/* .
+      - name: Move SDK in test folder
+        run: mkdir -p ./example/php/sdk &&  cp -r ./src/ ./example/php/sdk/
+      - name: Move SDK composer file in test folder
+        run: cp ./composer.json ./example/php/sdk/composer.json
+      - name: New config file for local project
+        run: cp ./composer-test.json ./example/php/composer.json

```

Comment:
I'm feeling this is a bit complex, and that we should not need the `compose-test` thing.

Do you think we could create an example folder inside the SDK and make tests on it? So that you only have to start the backend, without copying all the stuff, and people could play with an example directly in the PHP SDK?

Reply:
But you have two suggests:
- Simplify CI ( We can request prod backend server, to avoid rebuild in local the root repo)
- Add local example to help user to test 
So we suppose that user have already started the backend from their end ? Or with a example global Key from prod server

Reply:
> Simplify CI ( We can request prod backend server, to avoid rebuild in local the root repo)

we won't do that since it's not consistent (we can remove the flag for instance)

> Add local example to help user to test

We can suggest them to run the docker-compose thing from the main repo like:

---

In order to run the project, do the following:

```sh
$ git clone https://root-repo && cd root-repo
$ docker-compose up
$ cd ../sdk-php && composer install && php -S localhost:3000
```

Reply:
more explicit for sure, but the idea would be this